handbook/benefits.md

```diff
@@ -53,26 +78,69 @@ The following stipend amounts have been developed to help cover some of the cost
 
 ### Receiving the Benefits Stipend
 
-The benefits stipend will automatically be added to your monthly payroll. You do not need to request for a reimbursement through Airbase.
+The benefits stipend will automatically be added to your monthly payroll. You do not need to request a reimbursement through Airbase.
 
 ### **Retirement & Pension Plans**
 
 As previously mentioned we look to target locally relevant and competitive benefit options and retirement and pension plans fall under the same philosophy. We are currently working towards getting each required Pension scheme and retirement plan solidified. More details will be forthcoming.
```

Comment:
Change "**P**ension" to "**p**ension"

Reply:
Should we add "and will be updated here once we have more information" or something along those lines, like you have in the distributed experience section here https://github.com/dbt-labs/corp/blob/new-handbook-launch/handbook/about_dbt_labs.md#-how-is-distributed-work-actually-conducted